### ISSUE TITLE
Does the repository continue to update localized content?

### ISSUE THREAD

**Issue by weiyou2012** [ProxymanApp/proxyman-localization#15]
I found that some Chinese content is not included in the latest version of the App. I wonder if ProxymanApp still accepts updates from this repository.

**Comment by NghiaTranUIT** [comment]
yes, it doesn't includes in the latest build because I'm looking for an alternative solution to implement the Localization in the app. Maybe 3rd-party service to make it easier to contribute

**Comment by zigad** [comment]
Hey @NghiaTranUIT did you check out https://hosted.weblate.org?

Maybe it could help. I have translated 2 open source projects so far and would love to help with translating Proxyman

**Comment by NghiaTranUIT** [comment]
@zigad not really. I plan to use AI Localization tool to help me in future update

**Comment by NghiaTranUIT** [comment]
new update 👍
